runtime/src/accounts_db.rs

```diff
@@ -5405,17 +5456,19 @@ impl AccountsDb {
         purged_slot_pubkeys: HashSet<(Slot, Pubkey)>,
         pubkey_to_slot_set: Vec<(Pubkey, Slot)>,
         is_dead: bool,
+        pubkeys_removed_from_accounts_index: &PubkeysRemovedFromAccountsIndex,
```

Comment:
`remove_dead_slots_metadata` is also called from `process_dead_slots`, so I think that one is correct to have the new arg.

Reply:
yes, because purging things from the cache will never subsequently 'unref', so the return is irrelevant. Things in the write cache don't get a refcount.

Reply:
That works. I was just imagining I'd elevate it through the functions that end up needing to pass something. Just in case someone changes somethign later.